.github/workflows/prettier.yml

```diff
@@ -0,0 +1,26 @@
+name: Check code format
+
+# This action works with pull requests and pushes
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
+        with:
+          # This part is also where you can pass other options, for example:
+          dry: True
+          prettier_options: --check "**/*.{js,md,yml,css,html}"
```

Comment:
are these options redundant?
And I see that you added `yml` and other files extensiosn, which maybe answer my other question

Reply:
i think @ctmbl meant `dry: True` and `--check` might be redundant?

Reply:
Yes indeed, I removed it :heavy_check_mark:

Reply:
That was it tks!

Reply:
Indeed I prefer to use raw prettier argument to pass to it, instead of the [creyD prettier_action](https://github.com/creyD/prettier_action). Tks for [6087ce7](https://github.com/iScsc/iscsc.fr/pull/83/commits/6087ce7680edc414a6be1634101d1a2dd7e33306)